src/jupyter_contrib_nbextensions/nbextensions/toc2/main.css

```diff
@@ -145,9 +149,18 @@ padding-left: 20px;
 
 /* don't waste so much screen space... */
 
-.toc .toc-item .toc-item {
+/*.toc .toc-item .toc-item {
   padding-left: 1em;
 }
+*/
+.toc-item ul, li { margin:0; padding:0 }
+.toc-item li > span { display:block }
+.toc-item li > span { padding-left:0em }
+.toc-item li li > span { padding-left:1em }
+.toc-item li li li > span { padding-left:2em }
+.toc-item li li li li > span { padding-left:3em }
+.toc-item li li li li li > span { padding-left:4em }
+.toc-item li li li li li li > span { padding-left:5em }
```

Comment:
I wanted to highlight the entire line containning the section item on hover. 
I used the answer [here](https://stackoverflow.com/questions/9537444/highlight-entire-line-in-a-nested-list-on-hover) and [here](http://jsfiddle.net/dYdQB/). There might be a better way to do it but didn't figured it. 

- ok for `.toc-item li`